<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyInvestment</class>
 <widget class="QDialog" name="MyInvestment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="showInvestment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>861</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ваши инвестиции</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sell">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>480</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Продать</string>
   </property>
  </widget>
  <widget class="QComboBox" name="chooseInvestments">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>461</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="chooseCount">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backToCabinet">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>520</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Личный кабинет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>480</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
